native/components/PortfolioCard/SinglePortfolioProject.jsx

```diff
@@ -1,7 +1,8 @@
 import { useState, useEffect } from 'react';
 import uuid from 'react-native-uuid';
-import { handleTextChange } from '../../services/utils/handlers';
 import { Button, Text, TextInput, View } from 'react-native';
+import { handleTextChange, handleToggle } from '../../services/utils/handlers';
+import { useSelector } from 'react-redux';
```

Comment:
move package to top section of imports, components second, assets like handlers/stylesheets come last